pkg/acsengine/defaults.go

```diff
@@ -733,10 +733,8 @@ func setAgentNetworkDefaults(a *api.Properties) {
 
 			// Allocate IP addresses for pods if VNET integration is enabled.
 			if a.OrchestratorProfile.IsAzureCNI() {
-				if a.OrchestratorProfile.OrchestratorType == api.Kubernetes {
```

Comment:
This was redundant because `IsAzureCNI()` returns false if `a.OrchestratorProfile.OrchestratorType != api.Kubernetes`